src/components/PetList.js

```diff
@@ -55,7 +68,54 @@ export default function PetList(props) {
     { value: 'houseTrained', label: 'House Trained' },
   ];
 
+  const savePreferences = async (e) => {
+    e.preventDefault();
+    const objectKeys = ['typeOfAnimal', 'breed', 'sex', 'minAge', 'maxAge', 'more'];
+    const preferenceTypes = [typeOfAnimal, breed, sex.value, minAge, maxAge, more];
```

Comment:
I think `sex.value` might cause null exception when there is no sex assigned.

Reply:
Good catch!